SD DB Setup.sql

```diff
@@ -422,7 +422,7 @@ create table voting_rights
     voting_notes text,
     pending_leg text,
     date_modified timestamp not null default current_timestamp(),
-    user_modified varchar(20) not null);
+    user_modified varchar(20) default null);
     
```

Comment:
good point since we've set a trigger to set the current user up anyway. @LinneaHarts i think it would make sense to make all the user_modified columns nullable in this script to be consistent..thoughts?